src/DataFramesMeta.jl

```diff
@@ -163,15 +175,14 @@ function with_helper(d, body)
     membernames = Dict{Any, Symbol}()
     funname = gensym()
     body = replace_syms!(body, membernames)
-    if isempty(membernames)
```

Comment:
Isn't this branch relevant anymore?

Reply:
I don't think so. That code is pre 0.6. My guess is splatting empty collections wasn't allowed back then or something. All tests pass, even ones with empty arguments. 

Plus, this branch is a source of inconsistent behavior, see #184